repo/packages/B/beta-tensorflow/1/marathon.json.mustache

```diff
@@ -0,0 +1,131 @@
+{
+  "id": "{{service.name}}",
+  "cpus": 1.0,
+  "mem": 1024,
+  "instances": 1,
+  "cmd": "export LD_LIBRARY_PATH=$MESOS_SANDBOX/libmesos-bundle/lib:$LD_LIBRARY_PATH; export MESOS_NATIVE_JAVA_LIBRARY=$(ls $MESOS_SANDBOX/libmesos-bundle/lib/libmesos-*.so); export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jre*/); export JAVA_HOME=${JAVA_HOME%/}; export PATH=$(ls -d $JAVA_HOME/bin):$PATH &&  export JAVA_OPTS=\"-Xms256M -Xmx512M -XX:-HeapDumpOnOutOfMemoryError\" &&  ./tensorflow-scheduler/bin/tensorflow ./tensorflow-scheduler/svc.yml",
+  "labels": {
+    "DCOS_COMMONS_API_VERSION": "v1",
+    "DCOS_COMMONS_UNINSTALL": "true",
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{service.name}}",
+    "MARATHON_SINGLE_INSTANCE_APP": "true",
+    "DCOS_SERVICE_NAME": "{{service.name}}",
+    "DCOS_SERVICE_PORT_INDEX": "0",
+    "DCOS_SERVICE_SCHEME": "http"
+  },
+  {{#service.service_account_secret}}
+  "secrets": {
+    "serviceCredential": {
+      "source": "{{service.service_account_secret}}"
+    }
+  },
+  {{/service.service_account_secret}}
+  "env": {
+    "BOOTSTRAP_URI": "{{resource.assets.uris.bootstrap-zip}}",
+    "CONFIG_TEMPLATE_PATH": "tensorflow-scheduler",
+
+    "FRAMEWORK_NAME": "{{service.name}}",
+    "FRAMEWORK_LOG_LEVEL": "{{service.log_level}}",
+    "FRAMEWORK_USER": "{{service.user}}",
+    "FRAMEWORK_PRINCIPAL": "{{service.service_account}}",
+    "TF_IMAGE": "{{service.tf_image}}",
+    "GPU_TF_IMAGE": "{{service.gpu_tf_image}}",
+    {{#service.job_url}}
+    "JOB_URL": "{{service.job_url}}",
+    "JOB_PATH": "{{service.job_path}}",
+    "JOB_NAME": "{{service.job_name}}",
+    {{/service.job_url}}
+    {{^service.job_url}}
+    "JOB_URL": "{{resource.assets.uris.examples-zip}}",
+    "JOB_PATH": "tensorflow-examples",
```

Comment:
Just curioys, why is the URL a variable, and the PATH not?

Reply:
This is currently used to trigger the CI tests. Not that `JOB_URL` is not a variable, it is just populated from the `resource.json` file and the full URI is determined at build time.